src/frontend/templates/components/Navbar.js

```diff
@@ -1,8 +1,19 @@
 import React, { useState } from 'react';
 import '../../styles/components/navbar.css';
-import { Link } from 'react-router-dom';
+import { Link , useNavigate } from 'react-router-dom';
+import { useUser } from '../../contexts/userContext';
+import { toastSuccess } from '../../services/toastService';
 
 export const Navbar = () => {
+  const { userState , userDispatcher } = useUser();
+  const navigate = useNavigate();
+
+  const logoutUser = async (e) => {
```

Comment:
e is passed but not ussed. this creates a little confusion.

Reply:
hi shivam ,sure will remove e in cases its not used